### ISSUE TITLE
Populate Team dropdown on the Activity page between seasons

### ISSUE THREAD

**Issue by ArushiSinghal** [rails-girls-summer-of-code/rgsoc-teams#934]
On the Activity page, the "Team drop-down" is of no use.
![screenshot-2018-2-11 rgsoc - teams app](https://user-images.githubusercontent.com/20858804/36079324-d9f06926-0fa7-11e8-9d48-3245c435568d.png)

Edit: Summing up the discussion below. The requirements are:
1) During the `current season`, the dropdown is populated with the teams that are `accepted` in the current season
2) Until the `current season` starts, the dropdown is populated with the teams that were `accepted` in the previous season. 

Updated the issue title accordingly.

**Comment by ramonh** [comment]
Hey @ArushiSinghal thanks for pointing this out!

The purpose of this dropdown is to display the accepted teams of the current Summer of Code season. 

So as you correctly found, the dropdown is empty between seasons. 

One thing we could do is hide the dropdown if there are no teams to display and show it otherwise.

What do you think?

**Comment by ArushiSinghal** [comment]
Yes, we should hide the dropdown when there is no team. @ramonh  should I go ahead to solve this issue?

**Comment by ramonh** [comment]
Sounds good, thanks @ArushiSinghal go for it!

**Comment by carpodaster** [comment]
We could also fill the menu with the selected teams of the most recent season when we don't have selected teams for the current one, yet. Wdyt?

**Comment by ramonh** [comment]
Good point, this is much better!
